ui-components/src/locales/general.json

```diff
@@ -1327,6 +1327,7 @@
     "unitType": "Unit Type",
     "url": "URL",
     "view": "View",
+    "viewListings": "View Listings",
```

Comment:
Not sure why, but it looks like this text isn't making it in:

![image](https://user-images.githubusercontent.com/8754454/135118174-5f52c5d4-a02a-4f32-ab1b-373752472e14.png)